ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,295 @@
+import pytest
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from product.models import Product
+from django.core.exceptions import ObjectDoesNotExist
+
+
+@pytest.fixture
+def valid_quantity():
+    return '5'
+
+
+@pytest.fixture
+def supplier_product0_id(supplier_product0):
```

Comment:
Since you have supplier_product0 you can have its id why do you need to make a fixture for that